Refactor storage stats calculation to use powers of 2 for more accurate values

skrashevich opened the pull request.

skrashevich committed:
"Refactor storage stats calculation to use powers of 2 for more accurate values"

NickM-27 commented:
The function returns in bytes, if power of 2 is going to be used then the frontend should be updated to show the unit as `MiB` and `GiB`, etc. instead of `MB`, `GB`, etc.

skrashevich committed:
replace 1000000 to 2^20

skrashevich committed:
Refactor storage unit size display to use binary prefixes

This commit updates the display of storage unit sizes in both the camera storage stats and the Storage component in the web UI to use binary prefixes (MiB and GiB) instead of decimal prefixes (MB and GB). This provides more accurate and consistent representation of storage sizes

skrashevich commented:
> The function returns in bytes, if power of 2 is going to be used then the frontend should be updated to show the unit as `MiB` and `GiB`, etc. instead of `MB`, `GB`, etc.

Khm. Looks like as regional terminology difference :) but in `getUnitSize` you was dividing to 1024, but using MB notation

NickM-27 commented:
Thanks, that makes sense. That wasn't my code but yeah that was incorrect.
